bin/run_demo.sh

```diff
@@ -169,7 +169,11 @@ test_quantum_butler() {
   done
 
   # Bring home the datasets, --update-output-chain also creates output chain
-  # collection from metadata stored in a graph.
+  # collection from metadata stored in a graph. Ingest some via a zip file.
+  output=$(butler --log-level=VERBOSE --long-log zip-from-graph "$graph_file" DATA_REPO ./ -d "cal*")
+  echo $output
+  zip_path=$(echo $output | grep .zip | awk -F/ {'print $NF'})
```

Comment:
Maybe `fgrep .zip`? Or even `egrep "[.]zip$"`? I trust that printed path will always have `/` in it even for the file in the `$PWD`?

Reply:
Yes, there is always a `/` because a full `file` URI is printed.